src/redux/utils.ts

```diff
@@ -114,15 +114,17 @@ export function errorMsgResponse(response: any) {
   } else {
     for (let key in response.data) {
       errorMsgResponse.push(
+        errorMsgResponse["__detail__"] = response.data.detail,
```

Comment:
This line should be outside the for loop.

Reply:
Also, we should probably get rid of the `if`...`else` altogether. Maybe something like:

```ts
if (response.data && Object.prototype.hasOwnProperty.call(response.data, "detail")) { 
  errorMsgResponse["__detail__"] = response.data.detail;
}

for (let key in response.data) {
  if (key === "__detail__") continue;
  errorMsgResponse[key] =
    // code goes here
}
```

Reply:
We should also change the type of `errorMsgResponse` to:

```ts
const errorMsgResponse: {[key: string]: string};
```